### PR TITLE
appveyor: Add -l test_suite to .appveyor.yaml

### PR DESCRIPTION
**Pull request by ernestognw**
As requested on #16976

No breaking changes

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -60,7 +60,7 @@ after_build:
 - ps:  clcache -z
 #- 7z a bitcoin-%APPVEYOR_BUILD_VERSION%.zip %APPVEYOR_BUILD_FOLDER%\build_msvc\%platform%\%configuration%\*.exe
 test_script:
-- cmd: src\test_bitcoin.exe -k stdout -e stdout 2> NUL
+- cmd: src\test_bitcoin.exe -l test_suite -k stdout -e stdout 2> NUL
```
Comment (MarcoFalke) [inline .appveyor.yml]: that is going to print the boost test messages always, even if everything passes

I am not sure about any log limits for appveyor, but this seems a bit verbose to me

Comment (MarcoFalke) [inline .appveyor.yml]: Also, it would be nice to run the tests in parallel (like we do with automake `make check -j xx`)

**Comment by MarcoFalke** [comment]
re-run ci

**Comment by MarcoFalke** [comment]
Doesn't work (nothing is printed):

```
src\test_bitcoin.exe -l test_suite -k stdout -e stdout 2> NUL
Command exited with code -1073741819

**Comment by MarcoFalke** [comment]
cc @sipsorcery

**Comment by sipsorcery** [comment]
Looks like #17573 may be causing test_bitcoin.exe to seemingly do nothing. Currently tracking down why.

**Comment by sipsorcery** [comment]
Running the `test_bitcoin.exe` with the addition of `-l test_suite` adds just under 1000 lines of logs to the appveyor output. I suspect it has a negligible impact on the appveyor job performance.

My concern would be for contributors needing to look at the logs the burden is increased. I don't think it gives us a lot of bang for our buck as most of the failures are at the build stage or the python functional tests. If a way could be found to only output the logs if there's a `test_bitcoin.exe` failure that would be useful.

**Comment by MarcoFalke** [comment]
re-run ci

File: .appveyor.yml (hunk 1 above)
Comment (MarcoFalke) [reply .appveyor.yml]: Jup, it does print it unconditionally: https://ci.appveyor.com/project/DrahtBot/bitcoin/builds/29582061#L1000

**Comment by fanquake** [comment]
Going to close. @sipsorcery maybe you'd like to pick this up if you think it's an improvement.

**Pull request closed without merging**